src/integrationTest/java/org/opensearch/test/framework/cluster/LocalCluster.java

```diff
@@ -297,6 +297,16 @@ private static void triggerConfigurationReload(Client client) {
         }
     }
 
+    public void triggerConfigurationReloadForSingleCType(Client client, CType cType, boolean ignoreFailures) {
```

Comment:
This is a test method. I will look into refactoring it, but if set to `true` then this method won't throw a RuntimeException and otherwise would throw a RuntimeException. This is to enable repeatedly running this method during node bootstrap to simulate the error scenario.